packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -90,6 +92,13 @@ class ImageCache {
   int get currentSizeBytes => _currentSizeBytes;
   int _currentSizeBytes = 0;
 
+  /// Callback that is executed when inserting an image that is too large for
+  /// the cache.  Set to `null` for the default behavior.
```

Comment:
Can we describe the behavior of the default handler here?

Reply:
Done.